lichess-bot.py

```diff
@@ -712,4 +711,4 @@ def intro():
         engine_factory = partial(engine_wrapper.create_engine, CONFIG)
         start(li, user_profile, engine_factory, CONFIG, logging_level, args.logfile)
     else:
-        logger.error("{} is not a bot account. Please upgrade it to a bot account!".format(user_profile["username"]))
+        logger.error(f'{user_profile["username"]} is not a bot account. Please upgrade it to a bot account!')
```

Comment:
If you use `username` instead of `user_profile["username"]`, then you can use double quotes on the whole string.